browser-extensions/common/js/content-scripts/content-script-parkrunner-summary.js

```diff
@@ -0,0 +1,39 @@
+function get_challenge_link_location() {
+    return $("div[id=content]").find("p:first")
+}
+
+function get_athlete_id() {
+    // Very basic method to get only the parameter we care about
+    var page_parameters = window.location.search
+    var athlete_id = undefined
+
+    if (window.location.pathname.startsWith('/parkrunner')) {
+        athlete_id = window.location.pathname.match('parkrunner\/([0-9]+)\/')[1]
+    } else if (page_parameters.includes('athleteNumber=')) {
+        athlete_id = page_parameters.split('athleteNumber=')[1].split('&')[0]
+    }
+
+    console.log('Athlete ID: '+athlete_id)
+    return athlete_id
+}
+
+
+var athlete_id = get_athlete_id()
+if (athlete_id != null) {
```

Comment:
this needs to be `!== undefined`, as you previously changed to standardising on returning `undefined`. I think I can edit it, let me give that a go